managed/services/management/postgresql.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/percona/pmm/managed/services"
 )
 
+const (
+	defaultAutoDiscoveryDatabaseLimit = 1000
```

Comment:
let's have it low like 10 or 15

Reply:
@BupycHuk Should Matěj change this also on FE? Default 1000 is also there (like in case of MySQL).

Reply:
Yes